src/Extensions/Hosting/Certificates/InitializationCertificateReader.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Omex.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Hosting.Certificates
+{
+	/// <summary>
+	/// CertificateReader to use until DI container build.
+	/// </summary>
+	/// <remarks>
+	/// Please prefer using instance resolved from DI container.
+	/// </remarks>
+	public class InitializationCertificateReader
```

Comment:
Could it be a static class?

Reply:
oh yeah it should, missed it, thanks, good catch

Reply:
fixed